src/components/item/item-sliding.scss

```diff
@@ -29,14 +28,32 @@ ion-item-options {
   height: 100%;
 
   font-size: 14px;
-
   visibility: hidden;
+
+  right: 0;
+
+  @include rtl() {
+    justify-content: flex-start;
+
+    &:not([side=right]) {
```

Comment:
these `:not([side=right])` are here for default side (when side is not defined in sliding option item) in RTL mode.
In LTR mode there should be positioned on right by default but on RTL they need to be placed left...
In another word when side is not defined it should be placed on `end`